src/codec/RowWriterV2.cpp

```diff
@@ -637,6 +643,7 @@ WriteResult RowWriterV2::write(ssize_t index, folly::StringPiece v) noexcept {
   auto field = schema_->field(index);
   auto offset = headerLen_ + numNullBytes_ + field->offset();
   switch (field->type()) {
+    case meta::cpp2::PropertyType::GEOGRAPHY:  // write wkb
```

Comment:
will convert wkt to wkb ?

Reply:
The converter is not happening here. When construing Geography value, the input wkt has been converted to wkb.
This notes `// write wkb` is just to emphasize that WKB is stored for Geography.
See the Geography values's data member:
```
struct Geography {
std::string wkb;
};
```